api.go

```diff
@@ -174,46 +199,56 @@ func Build(buildServerPath string, dockerInfo DockerInfo) string {
 	}
 }
 
-func rsyncProject(buildServerPath, remoteLoc string) {
+func rsyncProject(ctx log.Interface, buildServerPath, remoteLoc string) {
 	rsyncCommand := exec.Command("scripts/rsync_project.sh", buildServerPath, remoteLoc)
 	res, err := rsyncCommand.CombinedOutput()
-	logging.LogToFile(err)
-	logging.LogToFile(string(res))
+	if err != nil {
+		ctx.WithError(err)
+	}
+	ctx.Info(string(res))
 }
 
 func createTempSSHKey(dockerInfo DockerInfo, buildServerPath string) error {
 	var err_return error
+	ctx := log.WithFields(log.Fields{
+		"repo": dockerInfo.REPO_NAME,
+		"env":  dockerInfo.ENV_ID,
+	})
 
 	d1 := []byte(dockerInfo.DEPLOY_KEY)
 	err := ioutil.WriteFile(buildServerPath+"/id_rsa", d1, 0644)
-	logging.LogToFile(err)
+	if err != nil {
```

Comment:
So I had to add the error checking back. I kind of liked having it as single line as a function.

One think I could do is keep the logging package and replace what we had with apex logging stuff. I don't know if that is weird or not